src/main/g8/src/main/scala/$package$/args/Args.scala

```diff
@@ -1,44 +1,46 @@
 package $package$.args
 
```

Comment:
If you go by the github "lines changed" metric, it appears that the `decline` implementation is ever-so-slightly (+2 lines) more verbose than the `scopt` implementation.

That being said, I think the fact that we no longer have to think about arguments as "state" that is mapped by each parsing success, it's actually a good deal more simplistic.

 The obvious downside to this change is the arg-parser is now more restricted in what you can do. And the obvious upside of that downside is that the arg-parser is more restricted in what you can do. :joy:

Reply:
More restriction, more better.  With what you've shown me here with Decline, I see very little downside as far as the programmatic interface and specification goes.  The only "gotcha" is that it plays well with cats, and makes that be a requirement.  This could be a dealbreaker for some, but I'm pretty stoked on it.